spec/models/tslr_claim_spec.rb

```diff
@@ -16,6 +16,7 @@
     it "validates that the value is in the correct format" do
       expect(build(:tslr_claim, email_address: "notan email@address.com")).not_to be_valid
       expect(build(:tslr_claim, email_address: "name@example.com")).to be_valid
+      expect(build(:tslr_claim, email_address: "")).to be_valid
```

Comment:
Actually, do we need another test to check `(build(:tslr_claim, email_address: nil)` is valid? (i.e. the default) I'm sure it is, but we probably need to check this assumption as if it isn't, the whole flow will break.

Reply:
I felt like we had enough coverage for that case in the validation contexts?

https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/801f43202962a1a42dbc9a176f74898ad93c4866/spec/models/tslr_claim_spec.rb#L252

Reply:
Ah, true dat. Carry on then :)